pkgs/applications/misc/visidata/default.nix

```diff
@@ -130,6 +130,10 @@ buildPythonApplication rec {
     bash dev/test.sh
     runHook postCheck
   '';
+  postInstall = ''
+    python dev/zsh-completion.py
+    install -Dm644 _${pname} -t $out/share/zsh/site-functions
```

Comment:
```suggestion
    install -Dm644 _visidata -t $out/share/zsh/site-functions
```

Reply:
Why? It has no difference.

Reply:
> Why? It has no difference.

`_visidata` is more clear. Avoids unnecessary abstraction. It's static. etc.